compiler/p/codegen/OMRAheadOfTimeCompile.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2016 IBM Corp. and others
+ * Copyright (c) 2000, 2020 IBM Corp. and others
```

Comment:
this looks like the only change here, so why are we updating the copyright header?

Reply:
Thank you for the comment, this change was purged from the history.